docs/quickstart.md

```diff
@@ -37,18 +37,24 @@ We will provision:
     az account set --name <subscription name>
     ```
 
-2. Run the bicep deployment script:
+2. Optional: Create a new resource group for the demo resources. Having a new group would make it easier to delete the resources afterwards.
```

Comment:
Should we add the following statement explicitly - Deleting the resource group would delete all resources created by this script?